x-pack/plugins/security_solution_serverless/public/upselling/pages/investigation_guide_upselling.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AppFeatureKey } from '@kbn/security-solution-plugin/common';
+import { i18n } from '@kbn/i18n';
+
+export const UPGRADE_INVESTIGATION_GUIDE = (productTypeRequired: AppFeatureKey) =>
+  i18n.translate('xpack.securitySolutionServerless.markdown.insight.upsell', {
+    defaultMessage: 'Upgrade to {productTypeRequired} make use of insights in investigation guides',
+    values: {
+      productTypeRequired,
+    },
+  });
+
+export const investigationGuideUpselling = (requiredPLI: AppFeatureKey): string =>
+  UPGRADE_INVESTIGATION_GUIDE(requiredPLI);
```

Comment:
We are passing the `requiredPLI` as the `productTypeRequired`, we need to call `useProductTypeByPLI(requiredPLI);`

Reply:
(and maybe rename `useProductTypeByPLI` to `getProductTypeByPLI`)

Reply:
Good catch!